### PR TITLE
gen sqw progress

### PR DESCRIPTION
**Pull request by abuts**
fixes Re #1663. Small addition makes working with gen_sqw so much nicer

### CODE REVIEW

**Comment by oerc0122** [comment]
Nothing in the description or title tells me what this change is actually doing.

**Review by oerc0122 — CHANGES_REQUESTED** [review]

File: herbert_core/classes/data_loaders/@rundata/private/gen_runfiles_.m [hunk 1]
```diff
@@ -247,10 +256,13 @@
 
 
 % Do we build runfiles from one, multiple or no par files?
+dot_string_length = 0;
+max_dot_string_length = 50;
```
[the remark this answers is not in the document]
Comment (abuts) [reply herbert_core/classes/data_loaders/@rundata/private/gen_runfiles_.m:260]: may be. There are similar stuff in parallel, but doing so globaly is a bit of a different job. 
This is not parallel and have no relation to it.  As it is small convenience change, its usfulness is justified by its simplicity.

File: herbert_core/classes/data_loaders/@rundata/private/gen_runfiles_.m [hunk 2]
```diff
@@ -306,6 +320,26 @@
...
     end
 end
+if print_progress_log
+    if ~cr_printed
+        fprintf('\n');
+    end
+    fprintf('*** Finished constructuion of %d %s objects\n',n_files,name_of_class);
+end
+
+function [log_length,cr_printed] = do_print_log(do_print,log_length,max_length)
```
[the remark this answers is not in the document]
Comment (abuts) [reply herbert_core/classes/data_loaders/@rundata/private/gen_runfiles_.m:330]: would agree with this

File: herbert_core/classes/data_loaders/@rundata/private/gen_runfiles_.m (hunk 1 above)
[the remark this answers is not in the document]
Comment (abuts) [reply herbert_core/classes/data_loaders/@rundata/private/gen_runfiles_.m:258]: Would agree on ticket in technical dept

**Pull request merged**